commitizen/cz/base.py

```diff
@@ -1,13 +1,29 @@
-from typing import Optional
+from typing import Optional, List, Tuple
 from abc import ABCMeta, abstractmethod
 
+from prompt_toolkit.styles import merge_styles, Style
+
 
 class BaseCommitizen(metaclass=ABCMeta):
     bump_pattern: Optional[str] = None
     bump_map: Optional[dict] = None
+    DEFAULT_STYLE_CONFIG: List[Tuple[str, str]] = [
+        ("qmark", "fg:#673ab7 bold"),
+        ("question", "bold"),
+        ("answer", "fg:#f44336 bold"),
```

Comment:
I would use `#FF9D00` or `#673ab7` for the answer. Whichever you choose, it should also be the same for: `qmark`, `pointer`, `highlighted`. What do you think?

Reply:
I have no preference on colors. I just change it to `#FF9D00`